daskernetes/core.py

```diff
@@ -38,8 +39,14 @@ def __init__(
 
         self.api = client.CoreV1Api()
 
+        if namespace is None:
+            namespace = _namespace_default()
+
+        if name is None:
+            name = 'dask-%s-%s' % (os.environ['USER'], str(uuid.uuid4())[:10])
```

Comment:
https://docs.python.org/3/library/getpass.html is probably a better bit to use here than calling $USER directly.